contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,109 @@
+  As active users of the engine, you are better equipped to identify bugs and other issues than anyone else.
+  To let us know about your findings, fill out `this bug report form <https://github.com/godotengine/godot/issues/new>` on our GitHub.
+  Make sure to include as much information as possible to ensure these issues can easily be reproduced by others.
+
+- **Test Development Versions**
+  While it is recommended to use the stable releases for your projects, you can help us test dev releases, betas, and release candidates
+  by opening a copy of your project in them and checking what problems this introduces or maybe even solves. 
+  Make sure to have a backup ready, since this can produce irreversible changes.
+
+  Find recent development versions `here <https://godotengine.org/download/preview/>`
+  
+- **Contribute Engine Code (mainly C++)**
+  The engine development is mainly coordinated on our `Contributor RocketChat <https://chat.godotengine.org/>`, 
+  so if you are serious about making PRs you should join us there!
+
+  Read more about the **technical submission process**: :ref:`doc_first_steps`
+
+  *Attention: this page is currently under re-contruction, and some teams might be more active than others*
+  For each subject area of the engine, there is a corresponding team to coordinate the work. 
+  Join the linked chat to get more eyes on your related PR, learn about open todos, or partake in meetings. 
+  For some areas, specialists might even be encouraged to step up as maintainer!
+  `List of teams <https://godotengine.org/teams/>`
+
+- **Review Code Contributions**
+  All pull requests need to be throroughly reviewed before they can be merged into the master branch.
+  Help us get a headstart by participating in the code review process.
+  .. add a howto/chat to join
+  
+  Read more about our **style guide and other requirements**: :ref:`doc_pr_review_guidelines`
+
+- **Write Plugins (GDScript, C#, & more)**
+  Community addons are not directly included in the core engine download or repository, yet they provide essential quality of life upgrades for your fellow game developers.
+  Upload your plugins to the `Godot Asset Library <https://godotengine.org/asset-library/asset>` to make them available to others.
+
+- **Demo projects (GDScript, C#, and making Assets)**
+  We provide new users with `demo projects <https://github.com/godotengine/godot-demo-projects/>` so they can quickly test new features or get familiar with the engine in the first place.
+  At industry events, we might even exhibit these demo projects to showcase what Godot can do! 
+  Help improve existing projects or supply your own to be added to the pool, and join the `Contributor RocketChat <https://chat.godotengine.org/channel/demo-content>` to talk about it.
+
+- **Documentation**
+  The documentation is one of the most essential parts of any tech project, yet the need to document new features and substential often gets overlooked. 
```

Comment:
not sure what `substantial` stands for in this context.

Reply:
substantial changes - this is why you dont use VSCode as a text editor, people 😅